src/__tests__/integration.ts

```diff
@@ -0,0 +1,54 @@
+import { createTestClient } from 'apollo-server-testing';
+import gql from 'graphql-tag';
+import { constructTestServer, mockedContext } from './__utils';
+
+
+
+
+describe('Integration', () => {
+  describe('Queries', () => {
+    it('users - read one by Id - success', async () => {
+      
+      const {server, geoDataStorage, storeDataStorage} = constructTestServer();
+
+      mockedContext.db.scan.mockReturnValueOnce({
+        promise: () => Promise.resolve(mockedReadByIdDbResponse)
+      });
+
+      const { query } = createTestClient(server);
```

Comment:
(continued) How do you decide when to add white spaces when object destructuring?

Reply:
Well, I always decide to use white spaces, but sometimes I miss it. And I relax on this becuase I always include the linting validations as part of CI and CD.

In this exercise, there are some files without the newline char becuase the linting corrections are pending (I put noted about it in my delivered documentations) I prefer don't to delay the delivery just because the linting was pending.